CMakeLists.txt

```diff
@@ -82,11 +120,12 @@ if (NOT STORM_USE_CLN_EA OR NOT STORM_USE_CLN_RF)
     set(PYCARL_IMPORTS "${PYCARL_IMPORTS}\nfrom stormpy.pycarl import gmp")
 endif()
 
-set(CARL_VERSION ${carl_VERSION})
 if ((USE_PARSER) AND (carlparser_FOUND))
     set(CARL_WITH_PARSER "True")
+    set(PYCARL_HAS_PARSE TRUE)
```

Comment:
Is this maybe something where we should use set_variable_string (to be consistent with the above?)

Reply:
Done.